client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -306,12 +320,22 @@ const Checkout = React.createClass( {
 	}
 } );
 
+function mapStateToProps( state ) {
+	const cards = getStoredCards( state );
+	const selectedSite = getSelectedSite( state );
+	const selectedSiteId = getSelectedSiteId( state );
+	const selectedSiteSlug = getSelectedSiteSlug( state );
```

Comment:
This is optional, but I usually avoid creating a variable unless it's being reused in the function